--- proto/BUILD.bazel
+++ proto/BUILD.bazel
@@ -1,4 +1,18 @@
 load("@io_bazel_rules_go//go:def.bzl", "go_library")
+load("@rules_proto//proto:defs.bzl", "proto_library")
+
+proto_library(
+    name = "attest_proto",
+    srcs = ["attest.proto"],
+    visibility = ["//visibility:public"],
+    deps = [":tpm_proto"],
+)
+
+proto_library(
+    name = "tpm_proto",
+    srcs = ["tpm.proto"],
+    visibility = ["//visibility:public"],
+)

 go_library(
     name = "proto",
